compiler/rustc_ty_utils/src/layout.rs

```diff
@@ -492,8 +481,7 @@ fn layout_of_uncached<'tcx>(
 enum SavedLocalEligibility {
     Unassigned,
     Assigned(VariantIdx),
-    // FIXME: Use newtype_index so we aren't wasting bytes
-    Ineligible(Option<u32>),
+    Ineligible(Option<FieldIdx>),
```

Comment:
Hooray for fixing `FIXME`s [from 2019](https://github.com/rust-lang/rust/commit/786875824ca6281b87bafb0edff099a3a05f73ae#diff-e1a56e3e84412d1c8b5a3ed263db2118530e43609c207334ba2e7973db20f3e3R646) 🙂